Twig/Extension/TextFormatterExtension.php

```diff
@@ -50,7 +52,7 @@ public function getTokenParsers()
     public function getFilters()
     {
         return array(
-            'format_text' => new \Twig_Filter_Method($this, 'transform'),
+            'format_text' => new \Twig_SimpleFilter($this, 'transform'),
```

Comment:
for this to work it would have to look like: \Twig_SimpleFilter('format_text', array($this, 'transform'))